docs/quickstarts/outlook-quickstart.md

```diff
@@ -34,7 +34,15 @@ You can create an Office Add-in by using the Yeoman generator for Office Add-ins
 
     - **Which Office client application would you like to support?** - `Outlook`
 
-    ![The prompts and answers for the Yeoman generator in a command line interface.](../images/yo-office-outlook-1.png)
+    - **Which manifest would you like to use?** - Choose either `unified manifest for Microsoft 365` or `XML manifest`
+
+    [!INCLUDE [Unified manifest value proposition and feedback request](../includes/unified-manifest-value-prop-feedback.md)]
+
+    Depending on your choice of manifest, the prompts and answers should look like one of the following.
+
+    ![The prompts and answers for the Yeoman generator when task pane, JavaScript, Outlook, and XML manifest are chosen.](../images/yo-office-outlook-xml-manifest-javascript.png)
+
+    ![The prompts and answers for the Yeoman generator when task pane, JavaScript, Outlook, and unified manifest are chosen.](../images/yo-office-outlook-json-manifest-javascript.png)
 
     After you complete the wizard, the generator will create the project and install supporting Node components.
 
```

Comment:
I can't leave a comment directly on the affected section, so adding this here.

Lines 61 - 66 ("Explore the project"): Include the file structure of an add-in project that uses the unified manifest (see [Explore the project](https://learn.microsoft.com/office/dev/add-ins/quickstarts/outlook-quickstart-json-manifest#explore-the-project)).

Line 68 ("Update the code"): We may need to add a note specifying that this section only applies to projects that use the XML manifest. I have a to-do to incorporate this JS code into the Outlook Yo Office template, but I might not be able to complete the work in time.

Reply:
I updated the **Explore the project** section.

The **Update the code** section applies regardless of the type of manifest that they chose. 

Darren and I have already updated the task pane project for JavaScript so that the user can choose either language with either type of manifest. I'm not clear on what your ToDo is, but I don't think it's needed.

Reply:
Thanks, Ricky.

Ah, I forgot that the change you and Darren worked on included language selection. Thanks for clearing that up. My task involves adding the JS code in the "Update the code" section of the Outlook quick start to the Outlook Yo Office template, so that the user can run the add-in right after creating it. This would align the Outlook quick start with the other quick starts, such as Excel.

Reply:
> Thanks, Ricky.
> 
> Ah, I forgot that the change you and Darren worked on included language selection. Thanks for clearing that up. My task involves adding the JS code in the "Update the code" section of the Outlook quick start to the Outlook Yo Office template, so that the user can run the add-in right after creating it. This would align the Outlook quick start with the other quick starts, such as Excel.

OK. I think I get it now. Step 3 of the "Update the code" section would be deleted because that code would be in the project when it is created.